src/device-registry/utils/create-health-tips.js

```diff
@@ -24,12 +25,19 @@ const createHealthTips = {
       const limit = parseInt(request.query.limit, 0);
       const skip = parseInt(request.query.skip, 0);
       const filter = generateFilter.tips(request);
+      const language = request.query.language;
+      let translatedHealthTips;
 
-      const responseFromListHealthTips = await HealthTipModel(tenant).list({
+      let responseFromListHealthTips = await HealthTipModel(tenant).list({
         filter,
         limit,
         skip,
       });
+      if (language !== undefined) {
```

Comment:
Now, this logic is easier to understand than the one inside the create-events util. :) @BenjaminSsempala